live-usb-creator/rhel7-livemedia-dev.ks

```diff
@@ -372,8 +372,12 @@ vim-common
 python3
 protobuf-python
 expect
+# Run the test vector generation script without an actual X server.
+# xvfb-run /tmp/subzero_expect.sh ./gui.jar
```

Comment:
I'm just curious: What makes `/temp/subzero_expect.sh ./gui.jar` not work?

Reply:
great question!
```
Exception in thread "main" java.awt.AWTError: Can't connect to X11 window server using ':0.0' as the value of the DISPLAY variable.
	at sun.awt.X11GraphicsEnvironment.initDisplay(Native Method)
	at sun.awt.X11GraphicsEnvironment.access$200(X11GraphicsEnvironment.java:65)
	at sun.awt.X11GraphicsEnvironment$1.run(X11GraphicsEnvironment.java:115)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.awt.X11GraphicsEnvironment.<clinit>(X11GraphicsEnvironment.java:74)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at java.awt.GraphicsEnvironment.createGE(GraphicsEnvironment.java:103)
	at java.awt.GraphicsEnvironment.getLocalGraphicsEnvironment(GraphicsEnvironment.java:82)
	at java.awt.Window.initGC(Window.java:475)
	at java.awt.Window.init(Window.java:495)
	at java.awt.Window.<init>(Window.java:537)
	at java.awt.Frame.<init>(Frame.java:420)
	at com.squareup.subzero.framebuffer.Framebuffer.createWindow(Framebuffer.java:86)
	at com.squareup.subzero.framebuffer.Framebuffer.<init>(Framebuffer.java:64)
	at com.squareup.subzero.SubzeroGui.interactive(SubzeroGui.java:137)
	at com.squareup.subzero.SubzeroGui.main(SubzeroGui.java:114)
send: spawn id exp7 not open
    while executing
```

Reply:
^ to get around this I had to try it on a gui based machine to confirm my suspicion.